pkg/test/monitor/test_monitor.go

```diff
@@ -14,15 +14,16 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// Package monitor provides a TestMonitorClient test class to perform client monitoring based on MonitorConnection call.
+// Package monitor provides a TestMonitorClient test class to perform client monitoring based on MonitorConnection call and tests for monitor sdk chain components
 package monitor
 
 import (
 	"context"
-	"github.com/networkservicemesh/sdk/pkg/tools/serialize"
 	"runtime"
 	"time"
 
+	"github.com/networkservicemesh/sdk/pkg/tools/serialize"
```

Comment:
Do we know why imports are moving around here?  More interestingly... do we know why linting isn't catching it?

Reply:
Nope, no idea.